ospatch/yum_update.go

```diff
@@ -108,7 +108,7 @@ func RunYumUpdate(ctx context.Context, opts ...YumUpdateOption) error {
 
 	var pkgNames []string
 	for _, pkg := range fPkgs {
-		pkgNames = append(pkgNames, pkg.Name)
+		pkgNames = append(pkgNames, pkg.NameWithArch)
```

Comment:
As i explained in a comment below it is not very hard to pass data required for construction NameWithArch in idiomatic way, i totally against adding fields that is required for one specific use case to structure that used across code base hugely

Reply:
Makes sense, updated the structure.